srtcore/channel.cpp

```diff
@@ -252,6 +252,21 @@ void CChannel::setUDPSockOpt()
       }
 #endif
 
+#ifdef SRT_ENABLE_BINDTODEVICE
+      if (m_BindToDevice != "")
+      {
+          if (m_BindAddr.family() != AF_INET)
+          {
+              LOGC(mglog.Error, log << "SRTO_BINDTODEVICE can only be set with AF_INET connections");
+              throw CUDTException(MJ_NOTSUP, MN_INVAL, 0);
+          }
+
+          if (0 != ::setsockopt(m_iSocket, SOL_SOCKET, SO_BINDTODEVICE, m_BindToDevice.c_str(), m_BindToDevice.size()))
```

Comment:
Joda style? :)

Reply:
This way it's clearer at the beginning that the condition is for failure.